app/views/topbanner.html.erb

```diff
@@ -0,0 +1,12 @@
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+</head>
+
+<section>
+  <div class="container">
+    <div class="jumbotron" align="middle">
+      <img src="http://res.cloudinary.com/dqaserzum/image/upload/v1490689989/Screen_Shot_2017-03-28_at_10.29.07_pv9ybs.png" height="125" alt="'Topp banner Taste of Code" />
+      <p class="lead" style="padding:10px">Beginner Coding Workshop</p>
```

Comment:
In the wireframe it also says 'Taste of Code' above 'Beginner Coding Workshops' please add that too :)